package.json

```diff
@@ -21,6 +22,10 @@
   "bugs": {
     "url": "http://github.com/nickclaw/alexa-ssml/issues"
   },
+  "files": [
+    "lib",
+    "src"
+  ],
```

Comment:
Conflicts with `.npmignore`, just remove the `src` entry from that file instead.